### ISSUE TITLE
use_batch_norm causes unreasonable low validation accuracy with inception nets

### ISSUE THREAD

**Issue by simsicon** [tensorflow/models#890]
Hi,

model: models/slim/nets/inception_v4.py

I was training inception model from scratch with my own dataset which contains two classes only, and I noticed that start to using batch normalization with use_batch_norm parameter, the validation accuracy was around 50% ~ 60% while training accuracy went to nearly 100%, it was almost just guessing. It looked like overfitting, and when I turned off use_batch_norm, the validation accuracy went to nearly 100% soon.

So my question is how exactly batch normalization might cause that? Any ideas on how to deal with it, I tried bunches of hyper-parameters of batch norm, it seems not helping.

Thanks

**Comment by jart** [comment]
Thanks for reaching out. We try to keep this issue tracker limited to bugs and feature requests. We recommend posting on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) for community-driven support.
